packages/sls-env/src/environment-config.ts

```diff
@@ -32,7 +32,13 @@ type EnvironmentConfig<H extends Handler<any, any, any>> = {
 }
 
 const defaultInvocationId = (): string => crypto.randomBytes(16).toString('hex')
-const defaultInvodationContext = (): Record<string, any> => ({})
 
-export type { EnvironmentConfig, InvocationIdConstructor, InvocationContextConstructor }
-export { defaultInvocationId, defaultInvodationContext }
+interface InvocationContext extends Record<string, any> {
+  awsRequestId?: string
+  sha?: string
+}
+
```

Comment:
IMO that needs to be generic and can't be specific anyone would be able to define the shape of the invocation context